operator/pkg/central/extensions/reconcile_admin_password.go

```diff
@@ -49,6 +51,7 @@ type reconcileAdminPasswordExtensionRun struct {
 	password           string
 	passwordSecretName string
 	autoGenerated      bool
+	hasCredentials     bool
```

Comment:
This name seems quite vague, but this field is set only in a very specific case, this seems fishy... 🤔

Reply:
It is dropped thanks to your suggestion 👍